smartcontractkit/external-adapters-js
# Add CLI Tool which generates a README template from package.json for a single EA

## Closes #19970

## Description
- Running `yarn generate:readme path/to/adapter` will generate a `README.md` file (or overwrite existing one) with populated title, version # and environment variables. Other sections are still templates to fill in.
- Running `yarn generate:readme packages/scripts/src/generate-readme/readme-test-adapter` will test generating a README from the `readme-test-adapter`.

## Changes

- Addition of `generate:readme script`
- Addition of `readme-test-adapter` to test script against



## Steps to Test
For now, the only test is to run the following and confirm that the generated README has no diff from the previous one:
`yarn && yarn setup && yarn generate:readme packages/scripts/src/generate-readme/readme-test-adapter`



## Quality Assurance
